tests/benchmarks/peripheral_load/src/common.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef PERIPHERAL_STRESS_TEST_H
+#define PERIPHERAL_STRESS_TEST_H
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+
+/* Thread status */
+extern atomic_t started_threads;
+extern atomic_t completed_threads;
+
+/* Accelerator readout via SPI: */
+#define ACCEL_THREAD_COUNT_MAX		(20)
+#define ACCEL_THREAD_OVERSAMPLING	(3)
+#define ACCEL_THREAD_STACKSIZE		(1024)
+#define ACCEL_THREAD_PRIORITY		(5)
+#define ACCEL_THREAD_SLEEP			(200)
```

Comment:
why not use Kconfig for each module?

Reply:
It was a "base" version for further manual testing. We checked if it is possible to break driver operation by:
- setting longer execution with xxx_THREAD_COUN_MAX,
- playing with threads priorities,
- setting short sleep times.
It was easier to have these parameters in a single file.